native/core/src/spaces.rs

```diff
@@ -210,16 +210,16 @@ impl CoreClient {
                 // local uri
                 let path = PathBuf::from(avatar_uri);
                 let guess = mime_guess::from_path(path.clone());
-                let content_type = guess.first().expect("MIME type should be given");
-                let buf = std::fs::read(path).expect("File should be read");
-                let upload_resp = client.media().upload(&content_type, buf).await?;
+                let content_type = guess.first().expect("don't know mime type");
+                let buf = std::fs::read(path).expect("file reading failed");
```

Comment:
let's switch those to `.context()?` pls.

Reply:
`CoreClient::create_acter_space()` returns `std::result::Result` not `anyhow::Result`.
So we can't use `anyhow::Context` here.

Reply:
but `std::fs::read(path)` returns a `std::result::Result<..>` ... why are we not just doing a `?` here?

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1252/commits/8d12c22301b0fc4e33875dd74b4305d8ccbcaed4